apps/protocol-frontend/src/components/HomeShell.tsx

```diff
@@ -3,13 +3,11 @@ import { Link } from 'react-router-dom';
 import { useAccount } from 'wagmi';
 import { Flex, Button, Text, VisuallyHidden, Heading } from '@chakra-ui/react';
 import ConnectWallet from '../components/ConnectWallet';
-import PageHeading from './PageHeading';
 import CreateUserForm from './CreateUserForm';
-
 import { useAuth } from '../contexts/AuthContext';
 import { useUser } from '../contexts/UserContext';
-import { GOVRN_MOTTO } from '../utils/constants';
 import { GovrnSpinner } from '@govrn/protocol-ui';
+import GovrnTextLogo from './GovrnTextLogo';
```

Comment:
(Arbitrary place for comment)

It seems like the user name text needs to be wrapped or shortened

![Screenshot from 2022-12-03 00-18-10](https://user-images.githubusercontent.com/12705146/205337667-4defc10e-372a-4793-aa94-3fe3d23eeb4f.png)

Reply:
The new colors for the dashboard seem really muted and can be tough to read against a white background
![Screenshot from 2022-12-03 00-21-04](https://user-images.githubusercontent.com/12705146/205338195-b5d993a4-3112-4b7e-ab9f-9eb2c94153dc.png)

Reply:
Ah good catch, I think I know why -- I included the 50 and 100 values for this whereas we previously didn't. The lightest tints may be too washed out

Reply:
Even with removing the 50 and 100 values, it's still really washed. The new palette seems to be much more washed out. I think I can resolve this by using a subset starting at the darker tints, like 300